doc/programming_guide/mic_aggregator/mic_aggregator.rst

```diff
@@ -1,3 +1,248 @@
 .. _sln_voice_mic_aggregator_programming_guide:
 
 .. include:: ../../../examples/mic_aggregator/README.rst
+
+
+Operation
+=========
+
+The design consists of a number of tasks connected via the xcore-ai silicon communication channels.
+The decimators in the microphone array are configured to produce a 48 kHz PCM output. 
```

Comment:
I can see that this application relies on `MyTwoStageDecimator` from the (as far as I can see) undocumented `par_decimator` demo in the lib_mic_array repo. 

Having one demo rely on another feels like spaghetti in the making. It would be better to upgrade `MyTwoStageDecimator` to be a proper part of lib_mic_array that has a useful name and is part of the `lib_mic_array` cmake target. Then modify both demos to use it properly.

I can see that isn't really the scope of this review though so just highlighting what I noticed.

Reply:
Sorry I missed this before - this is a very good point. I followed the design idiom that mic_array set but agree this would be a helpful update. Mic array in general is quite hard to follow (in my opinion) and anything that tidies that up is useful. I'll create an issue.

Reply:
https://github.com/xmos/lib_mic_array/issues/210